File: reporting-test-client/testClientWebUi/src/app/core/services/irrigation-event.ts
export class IrrigationEvent {
    journalId: number;
    bearing: number;
    direction: string;
    velocity: number;
    isPumpOn: false;
    scheduleId: number;
    puvitControllerId: number;
    createdDate: Date;
    displaySubstance: string;
    timeBetweenEvents: number;
}

File: reporting-test-client/IrrigationReportingWebApi/IrrigationReportingWebApi/Models/IrrigationEvent.cs
using System;
using System.Linq;

namespace Trimble.Ag.IrrigationReporting.IrrigationReportingWebApi.Models
{
	public class IrrigationEvent
	{
		public long JournalId { get; set; }
		public decimal Bearing { get; set; }
		public string Direction { get; set; }
		public double Velocity { get; set; }
		public bool? IsPumpOn { get; set; }
		public int ScheduleId { get; set; }
		public string Substance { get; set; }
		public int PivotControllerId { get; set; }
		public DateTime CreatedDate { get; set; }
		public string DisplaySubstance { get; set; }
		public double TimeBetweenEvents { get; internal set; }
	}
}
